public/locales/en/translations.json

```diff
@@ -27,10 +27,10 @@
 	"roadmap.organization.1": "The packages repository will have sets extracted to allow for Special Interest Groups to more easily manage their lifecycles. For example, `pythonPackages` will be moved to a separate repository that will be the reponsibility of a Python SIG if there is interest.",
 	"roadmap.organization.2": "Flakes will be standardized with its current implementation as a v0. While not ideal, the feature is used far too widely to be changed or removed without breaking the ecosystem. Instead, this v0 implementation will be enabled and future work for Flakes that addresses its shortcomings may be handled by a Flakes SIG.",
 	"roadmap.organization.3": "The `aux` CLI will be modified to provide more ergonomic management of packages and systems. Additional subcommands such as `aux system switch` and `aux system build` will be added to make onboarding and ongoing maintenance easier.",
-	"roadmap.infa.title": "Infastructure",
-	"roadmap.infa.1": "Now that the project has significantly diverged from upstream, we will need to provide our own Continuous Integration and Binary Cache services. Existing governance structures will be used to manage the adoption of these technologies.",
-	"roadmap.infa.2": "Documentation will be included as a part of the Continuous Integration process and will have Continuous Deployment configured to keep it up to date, making it easier for contributors to create educational materials.",
-	"roadmap.infa.3": "Search infrastructure for packages and options will also be created.",
+	"roadmap.infra.title": "Infastructure",
```

Comment:
I notice it still says infa here... it's not a rekey so hopefully this is easy to fix, but maybe we should still do so here

Reply:
lmao